components/MainNavigation/BottomTabNavigator./index.js

```diff
@@ -35,21 +36,23 @@ export default function BottomTabNavigator({ navigation, route }) {
           tabBarIcon: ({ focused }) => <TabBarIcon focused={focused} name="md-folder" />,
         }}
       />
-      <BottomTab.Screen
+      {/* <BottomTab.Screen
         name="Data_Analysis"
         component={DataAnalysis}
         options={{
           title: 'Data Analysis',
           tabBarIcon: ({ focused }) => <TabBarIcon focused={focused} name="md-analytics" />,
         }}
-      />
+      /> */}
     </BottomTab.Navigator>
 
   );
 }
 
 function getHeaderTitle(route) {
-  const routeName = route.state?.routes[route.state.index]?.name ?? INITIAL_ROUTE_NAME;
+  const routeName = getFocusedRouteNameFromRoute(route) ?? INITIAL_ROUTE_NAME;
```

Comment:
@jamccomb92 this fixes that stupid warning we get everytime we login into the app (the warning about accessing the state of the route)